CDCgov/cdc-open-viz
# Feature/dev 9503

## [Replace With Ticket Number]



## Testing Steps
Open Box plot, make you you have data set with multiseries.
1. We need to add more series to see results where box plot chart can render more series. used to be one only.
2. Box plot tooltip issue fixed when user enters custom measure value like "Maximum"
3. Updated "Total" measure into "Count" measure where we show  count of items instead of sum.




## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
